medmodels/data_synthesis/mtgan/model/samplers.pyi

```diff
@@ -0,0 +1,32 @@
+from typing import List, Tuple
+
+import torch
+
+from medmodels.data_synthesis.mtgan.model.loaders import MTGANDataset
+
+class ConceptSampleIterator:
+    concept: int
+    patient_indices: List[int]
+
+    current_index: int
+    length: int
+
+    def __init__(
+        self, concept: int, patients_indices: List[int], shuffle: bool = True
+    ) -> None: ...
+
+class MTGANDataSampler:
+    dataset: MTGANDataset
+    device: torch.device
+    size: int
+    concept_samples: List[ConceptSampleIterator]
+
+    def __init__(
+        self,
+        dataset: MTGANDataset,
+        concept_sample_map: List[ConceptSampleIterator],
```

Comment:
Is concept_sample_map the same thing as concept_samples in line 22?

Reply:
Yes, should be. Let's change it to both being the same